### PR TITLE
chore: packing slip msg display bold

### PR DESCRIPTION
**Pull request by shantanu-kadam-epi**

### CODE REVIEW

**Review by rvpasari — CHANGES_REQUESTED** [review]
Why are we converting this message to bold, without the actual package nos in the message?

**Comment by shantanu-kadam-epi** [comment]
Packing slip message should be in bold, it is suggested by Himanshu

**Review by hrwX — DISMISSED** [review]

File: erpnext/stock/doctype/packing_slip/packing_slip.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def validate_case_nos(self):
 		elif not self.to_case_no:
 			self.to_case_no = self.from_case_no
 		elif cint(self.from_case_no) > cint(self.to_case_no):
-			frappe.msgprint(_("'To Case No.' cannot be less than 'From Case No.'"),
+			frappe.msgprint(_("<b>To Package No.: {0}</b> cannot be less than <b>From Package No.: {1}</b>",[doc.to_case_no , doc.from_case_no]),
```
Comment (sahil28297) [inline erpnext/stock/doctype/packing_slip/packing_slip.py]:
```suggestion
			frappe.msgprint(_("<b>To Package No.: {0}</b> cannot be less than <b>From Package No.: {1}</b>").format(doc.to_case_no, doc.from_case_no),
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Comment by shantanu-kadam-epi** [comment]
image

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**